pkg/apis/core/types.go

```diff
@@ -2865,6 +2872,12 @@ type PodSecurityContext struct {
 	// If unset, the Kubelet will not modify the ownership and permissions of any volume.
 	// +optional
 	FSGroup *int64
+	// FSGroupChangePolicy defines behavior of changing ownership and permission of the volume
+	// before being exposed inside Pod. This field will only apply to
+	// volume types which support fsGroup based ownership(and permissions).
+	// It will have no effect on ephemeral volume types such as: secret, configmaps
+	// and emptydir.
+	FSGroupChangePolicy *PodFSGroupChangePolicy
```

Comment:
Behavior of nil / unset should be documented.

Reply:
fixed